src/api/app/components/notification_filter_component.html.haml

```diff
@@ -1,43 +1,91 @@
-                                               filter_item: { kind: 'comments' }, selected_filter: @selected_filter)
-  = render NotificationFilterLinkComponent.new(text: 'Requests', amount: @count['BsRequest'], icon: 'fas fa-code-pull-request',
-                                               filter_item: { kind: 'requests' }, selected_filter: @selected_filter)
-  = render NotificationFilterLinkComponent.new(text: 'Incoming Requests', amount: @count['incoming_requests'], icon: 'fas fa-code-pull-request',
-                                               filter_item: { kind: 'incoming_requests' }, selected_filter: @selected_filter)
-  = render NotificationFilterLinkComponent.new(text: 'Outgoing Requests', amount: @count['outgoing_requests'], icon: 'fas fa-code-pull-request',
-                                               filter_item: { kind: 'outgoing_requests' }, selected_filter: @selected_filter)
-  = render NotificationFilterLinkComponent.new(text: 'Roles Granted', amount: @count['relationships_created'], icon: 'fas fa-user-tag',
-                                               filter_item: { kind: 'relationships_created' }, selected_filter: @selected_filter)
-  = render NotificationFilterLinkComponent.new(text: 'Roles Revoked', amount: @count['relationships_deleted'], icon: 'fas fa-user-tag',
-                                               filter_item: { kind: 'relationships_deleted' }, selected_filter: @selected_filter)
-  = render NotificationFilterLinkComponent.new(text: 'Build Failures', amount: @count['build_failures'], icon: "fas fa-xmark text-danger",
-                                               filter_item: { kind: 'build_failures' }, selected_filter: @selected_filter)
-  - if ReportPolicy.new(@user, Report).notify?
-    = render NotificationFilterLinkComponent.new(text: 'Reports', amount: @count['reports'], icon: 'fas fa-flag',
-                                                filter_item: { kind: 'reports' }, selected_filter: @selected_filter,
-                                                )
-  = render NotificationFilterLinkComponent.new(text: 'Workflow Runs', amount: @count['workflow_runs'], icon: 'fas fa-book-open',
-                                               filter_item: { kind: 'workflow_runs' }, selected_filter: @selected_filter)
-  - if Flipper.enabled?(:content_moderation, @user)
-    = render NotificationFilterLinkComponent.new(text: 'Appealed Decisions', amount: @count['appealed_decisions'], icon: 'fas fa-hand',
-                                                 filter_item: { kind: 'appealed_decisions' }, selected_filter: @selected_filter)
+= form_for(:notification, url: my_notifications_path, method: :get) do |form|
+  .list-group.list-group-flush.mt-3.mb-2
+    %h6.px-3.py-2
+      %b State
+    .px-4.py-1
+      = render partial: 'webui/shared/check_box', locals: { label: 'Unread', amount: @count['unread'] || 0,
```

Comment:
Is there a way to get `@count['unread'] == nil`?

Reply:
I think the `|| 0` is redundant

Reply:
This is true, and it is valid for all the filter checkboxes here (Off-Topic: it is true for all the `workflow_run_filter_component` too)